src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -77,7 +77,7 @@ private static JsonObject createItemContext(Item item) {
 
     String yearCaptionsToken = String.join("; ", item.getYearCaption());
 
-    String captionsToken = JsonStringArrayHelper.toStream(item.getCopyNumbers())
+    String copyNumbersToken = JsonStringArrayHelper.toStream(item.getCopyNumbers())
```

Comment:
Personally, I might statically import this method to make this more succinct

```suggestion
    String copyNumbersToken = toStream(item.getCopyNumbers())
```

Reply:
Fix